codalab/model/worker_model.py

```diff
@@ -35,10 +37,13 @@ class WorkerModel(object):
        listen on these sockets for messages and send messages to these sockets.
     """
 
+    ACK = b'a'
+
     def __init__(self, engine, socket_dir, ws_server):
         self._engine = engine
         self._socket_dir = socket_dir
         self._ws_server = ws_server
+        self._server_secret = os.environ["CODALAB_SERVER_SECRET"]
```

Comment:
Maybe I shouldn't access an environment variable here? Is there a better way to inject the server secret?